example/celery+django/requirements.txt

```diff
@@ -1,11 +1,11 @@
-amqp==2.2.1
-billiard==3.5.0.3
-bugsnag==3.1.1
```

Comment:
Do we want to pin the bugsnag version? Feels like the example should get the latest features!